From: tangxiaofeng <tangxiaofeng@kylinos.cn>
Date: Mon, 16 Mar 2026 13:37:56 +0800
Subject: <short summary of the patch>
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

TODO: Put a short summary on the line above and replace this paragraph
with a longer explanation of this change. Complete the meta-information
with other relevant fields (see below for details). To make it easier, the
information below has been extracted from the changelog. Adjust it or drop
it.

python-pip (25.1.1+dfsg-1ok2) huanghe; urgency=medium

  * 禁用sphinx_issues.
---
 docs/html/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/html/conf.py b/docs/html/conf.py
index d736e29..a2e1784 100644
--- a/docs/html/conf.py
+++ b/docs/html/conf.py
@@ -33,7 +33,7 @@ if "man" not in tags:  # type: ignore[name-defined] # noqa: F821
             "sphinx_copybutton",
             "sphinx_inline_tabs",
             #"sphinxcontrib.towncrier",
-            "sphinx_issues",
+            #"sphinx_issues",
         ),
     )
 
